database/sql/headers.go

```diff
@@ -160,6 +168,20 @@ const (
 	sqlTipOfChainHeight = `SELECT MAX(height) FROM headers WHERE header_state = 'LONGEST_CHAIN'`
 
 	sqlVerifyHash = `SELECT hash FROM headers WHERE merkleroot = $1 AND height = $2 AND header_state = 'LONGEST_CHAIN'`
+
+	sqlGetHeadersHeight = `
+	SELECT COALESCE(MAX(height), 0) AS startHeight
```

Comment:
According to the documentation
https://wiki.bitcoinsv.io/index.php/Peer-To-Peer_Protocol#getblocks
`Note that it is allowed to send in fewer known hashes down to a minimum of just one hash. However, the purpose of the block locator object is to detect a wrong branch in the caller's main chain. If the peer detects that you are off the main chain, it will send in block hashes which are earlier than your last known block. So if you just send in your last known hash and it is off the main chain, the peer starts over at block #1.`
and
`For the block locator object in this packet, the same rules apply as for the getblocks packet.`